generators/app/templates/elm/element/_elm.json

```diff
@@ -1,26 +1,32 @@
 {
-  "name": "<%= name %>",
-  "type": "application",
-  "source-directories": ["src"],
-  "elm-version": "0.19.0",
-  "dependencies": {
-    "direct": {
-      "elm/browser": "1.0.1",
-      "elm/core": "1.0.2",
-      "elm/html": "1.0.0",
-      "elm/http": "2.0.0"
+    "name": "<%= name %>",
+    "type": "application",
+    "source-directories": [
+        "src"
+    ],
+    "elm-version": "0.19.0",
+    "dependencies": {
+        "direct": {
+            "elm/browser": "1.0.1",
+            "elm/core": "1.0.2",
+            "elm/html": "1.0.0",
+            "elm/http": "2.0.0",
+            "elm/json": "1.1.3"
```

Comment:
This file changed a lot because I had to make `elm/json` a direct dependency as a workaround for https://github.com/rtfeldman/node-test-runner/issues/323